config/helm/appmesh-controller/templates/pdb.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if .Values.podDisruptionBudget }}
 {{- if gt (int .Values.replicaCount) 1 }}
 kind: PodDisruptionBudget
-apiVersion: policy/v1beta1
+apiVersion: policy/v1
```

Comment:
Just a note: I'm ok taking this once the version change for helm is removed, but I'll add that we'll probably need to do some inter-version testing for this before doing a full release. These sorts of inter-version incompatibility issues often create problems, and we'd need to be confident we have complete guidance for compatibility before doing a general release.